src/components/CategoryNavContainer/CategoryNav/CategoryNav.scss

```diff
@@ -1,12 +1,89 @@
 @import '~style/variables';
 
+.categoryNav {
+  background-color: $color-white;
+  display: flex;
+  padding-left: 20px;
+  width: 100%;
+  white-space: nowrap;
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+  -ms-overflow-style: -ms-autohiding-scrollbar;
+
+  @include for-desktop-up {
+    padding-left: 30px;
+  }
+}
+
 .link {
+  height: 100%;
   display: inline-block;
-  padding: 10px;
+  padding: 12px 5px;
   text-decoration: none;
+
+  @include for-desktop-up {
+    padding: 24px 5px;
+  }
+
+  &:hover div {
+    color: $color-white;
+  }
+
+  div {
+    @include type-nav-horizontal;
+
+    padding: 0 24px;
+    border-radius: 100px;
+    color: $color-grey--700;
+    box-shadow: 0 0 4px 2px rgba(0, 0, 0, 0.1);
+
+    @include for-desktop-up {
+      box-shadow: initial;
+
+      &:hover {
+        box-shadow: 0 0 4px 2px rgba(0, 0, 0, 0.1);
+      }
+    }
+  }
+
+  &:hover,
+  &.link__active {
+    .overview {
+      border: 1px solid $color-grey--500;
+      background-color: $color-white;
+      color: $color-grey--700;
+    }
+
+    .jobs {
+      background-color: $color-jobs--600;
```

Comment:
You don't need the longhand `background-color`, `background` is fine and what's used elsewhere.